app/models/checkout.rb

```diff
@@ -44,4 +51,41 @@ def card_number
   scope :old, -> { where('checked_in_at IS NOT NULL') }
   scope :current, -> { where('checked_in_at IS NULL') }
 
-end
+  private
+
+  def notify
+    if (self.checked_in_at != nil)
```

Comment:
I see what you are saying here @ChaseBro, but if I'm not mistaken, as it stands after the check in is the only reason this should be updated. Given the proximity to Carnival, I think we should go with it and open a high priority issue / bug. Thoughts?